core/lib/generators/mno_enterprise/install/templates/config/settings.yml

```diff
@@ -24,3 +24,7 @@ developer:
 # Enable Reviews in the marketplace
 reviews:
   enabled: false
+
+marketplace:
+  comparison:
```

Comment:
@x4d3 this name came from the issue description. Please confirm that I need to use the `apps_comparison`, it's 5 min to change